docs/tutorials/cli-beginners-guide.rst

```diff
@@ -0,0 +1,155 @@
+=========================================
+The Beginner's Guide to the Orquestra CLI
+=========================================
+
+.. note::
+
+    This guide assumes that you have already installed the ``orquestra-sdk``. This can be confirmed by running
+
+    .. code-block::
+
+        orq -h
+
+    For installation instructions, see :doc:`Installing<installing-macos-linux>` or :doc:`Installing on Windows <installing-windows>`.
+
+Creating a Workflow Definition
+==============================
+
+In order to run a workflow, it must be defined in a python file. For this guide, we'll assume that we have created a file ``workflow_defs.py`` with the following contents:
```

Comment:
**question**: WDYT about keeping every sentence on a separate line? This improves git diffs but doesn't make a difference for the rendered page.